wallet/src/util/ibc-assets.ts

```diff
@@ -64,4 +64,67 @@ export const ibcAssets: IbcAssets = {
         'ibc/B1E6288B5A0224565D915D1F66716486F16D8A44BF33A9EC323DD6BA30764C35',
     },
   },
+  stOSMO: {
+    chainInfo: {
+      chainName: 'Stride',
+      chainId: 'stride-1',
+      rpc: 'https://stride-rpc.polkachu.com:443',
+      addressPrefix: 'stride',
+      explorerPath: 'stride',
+      gas: '100000',
+    },
+    deposit: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-148',
+      denom: 'stuosmo',
+    },
+    withdraw: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-59',
+      denom:
```

Comment:
it's hard to review whether all these values are correct. I end up looking at other repos. Let's just source from them https://github.com/Agoric/wallet-app/issues/155

Reply:
Were you able to find this denom in chain registry? I ended up just looking at a raw keplr txn to figure it out

Reply:
https://github.com/cosmology-tech/chain-registry/blob/14e096249ae904190d83615f569521e9ec02e118/packages/assets/src/mainnet/agoric.ts#L715-L725

but the ibc denoms don't match.